Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml

```diff
@@ -56,6 +56,7 @@
                     android:focusable="true"
                     app:body="@string/srs_self_test_body"
                     app:headline="@string/srs_self_test_title"
+                    app:iconTint="@color/whiteAlpha60"
```

Comment:
Shouldn't this be #17191A99? whiteAlpha60 is #99FFFFFF

Reply:
The color is #17191A and the tint is the whiteAlpha60. Putting #17191A99 as a color won't give you the same result.

Reply:
Color reference can be found in the custom view's xml file.